js/classic/backend.js

```diff
@@ -0,0 +1,237 @@
+(function(exports) {
+  'use strict';
+
+  exports.classic = exports.classic || {};
+  exports.classic.backend = exports.classic.backend || {};
+
+  var ui = exports.classic.ui;
+  var backend = exports.classic.backend;
+
+  backend.BluetoothManagerTab = function(tabsManager) {
+    this._tabsManager = tabsManager;
+    this._tab = new ui.Tab({}, 'BluetoothManager');
+    this._adapterBlocks = [];
+
+    this._tabsManager.addTab(this._tab);
+
+    var events = ['adapteradded', 'adapterremoved', 'attributechanged'];
+    for (var i = 0; i < events.length; ++i) {
+      navigator.mozBluetooth.addEventListener(
+          events[i], this._updateBlocks.bind(this));
+    }
```

Comment:
nit: You could avoid using `var i` and for loop by writing this:
```js
var that = this;
events.forEach(function(eventName) {
  navigator.mozBluetooth.addEventListener(eventName, that._updateBlocks.bind(that));
});
```
Though I don't think it is a problem. Just a suggestion.